subsys/fs/zms/zms.c

```diff
@@ -118,7 +118,8 @@ static inline size_t zms_al_size(struct zms_fs *fs, size_t len)
 	if (write_block_size <= 1U) {
 		return len;
 	}
-	return (len + (write_block_size - 1U)) & ~(write_block_size - 1U);
+
```

Comment:
Line 46 split `uint64_t addr, ate_addr;` into 2 lines
Same for line 235 `	size_t bytes_to_cmp, block_size;`
Same for line 264 `	size_t bytes_to_cmp, block_size;`
Same for line 288 `	size_t bytes_to_copy, block_size;`
Same for line 537 `	uint64_t data_end_addr, ate_end_addr;`
Same for line 575 `	struct zms_ate empty_ate, close_ate;`
Same for line 638 `	struct zms_ate close_ate, garbage_ate;`
Same for line 812 `	uint64_t wlk_prev_addr, wlk_addr;`
Same for line 855 `	struct zms_ate close_ate, gc_ate, wlk_ate, empty_ate;`
Same for line 856 `	uint64_t sec_addr, gc_addr, gc_prev_addr, wlk_addr, wlk_prev_addr, data_addr, stop_addr;`
Same for lines 1034, 1039, 1040, 1362, 1363, 1364, 1505, 1621, 1622, 1623
Move declaration on line 1689 to top of function
Remove empty line on 1291, 1620,

Reply:
@nordicjm is splitting the lines part of the Coding Guidelines ?

Reply:
I don't know if it is or not but it is in-keeping with the whole rest of the code base, newly added code should not just differ, not sure if @henrikbrixandersen or @kartben know if it's documented somewhere

Reply:
@nordicjm I changed the variables declaration to avoid spending much time discussing this even if I am not convinced that it is better this way.
But I am curious where this rule is written.
If this is a Coding Guideline from Zephyr it must be written clearly and not discovered through code reviews

Reply:
@nordicjm please verify the last revision then remove the change request label